k8s/solr/chart/solr/templates/application.yaml

```diff
@@ -0,0 +1,124 @@
+    description: |-
+      Apache Solr is an open-source enterprise-search platform, written in Java, from the Apache Lucene project.
+
+      # Support
+      Google does not offer support for this solution. However, community support is available on [StackOverflow](https://stackoverflow.com/questions/tagged/solr).
+    maintainers:
+    - name: Google Click to Deploy
+      url: https://cloud.google.com/solutions/#click-to-deploy
+    links:
+    - description: 'User Guide: Google Click to Deploy SolrCloud'
+      url: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/solr/README.md
+    {{- if not .Values.solr.standalone }}
+    - description: 'SolrCloud: Getting Started'
+      url: https://lucene.apache.org/solr/guide/8_6/getting-started-with-solrcloud.html
+    {{- end }}
+    - description: 'Solr Tutorials'
+      url: https://lucene.apache.org/solr/guide/8_6/getting-started.html
+    notes: |-
+      # Get authentication credentials for the cluster
+
+      ```shell
+      gcloud container clusters get-credentials [CLUSTER_NAME] --zone [CLUSTER_ZONE]
+      ```
+
+      where:
+      * `[CLUSTER_NAME]` is the name of the cluster for this app
+      * `[CLUSTER_ZONE]` is the zone in which the cluster is operating
+
+      For more information, read the [instructions for getting authentication credentials for the cluster](https://cloud.google.com/kubernetes-engine/docs/quickstart#get_authentication_credentials_for_the_cluster).
+
+      Make sure that all components are in `ready` status.
+
+      Deployed service of {{ template "solrMode" . }} is ClusterIP type, so you can reach to {{ template "solrMode" . }} Admin UI within a kubernetes cluster.
+
+      {{- if not .Values.solr.standalone }}
+      SolrCloud is configured with [Basic authentication plugin](https://lucene.apache.org/solr/guide/8_6/basic-authentication-plugin.html) beside [ZooKeeper ACL](https://lucene.apache.org/solr/guide/8_6/zookeeper-access-control.html).
+      You can check credentials from application page in console or you can print it by running below commands:
+
+      ```shell
+      # Get and print SolrCloud user credentials
+      SOLR_PASSWORD=$(kubectl get secret --namespace \
+      {{ .Release.Namespace }} {{ .Release.Name }}-solr-secret \
+      -o jsonpath="{.data.solr-password}" | base64 --decode)
+
+      echo "username: solr"
+      echo "password: ${SOLR_PASSWORD}"
+      ```
+      {{- else }}
+
+      Solr deployed as a standalone mode without any authentication enabled and for development and testing purposes.
+
+      {{- end }}
+
+      Forward {{ template "solrMode" . }} service to your machine.
+      ```shell
+      kubectl port-forward svc/{{ .Release.Name }}-solr-svc \
+        --namespace {{ .Release.Namespace }} 8983
+      ```
+      
+      Then visit [http://localhost:8983/solr](http://localhost:8983/solr) on your web browser and login with `solr` user credentials if necessary.
+  info:
+  - name: Application Name
```

Comment:
This is not necessary, you can remove. {{ .Release.Name}} will appear at top as the page title, close to the partner name and link